Jenkinsfile-Release

```diff
@@ -17,7 +17,7 @@ properties([
   ])
 ])
 
-node('ubuntu-zion-legacy') {
+node('ubuntu-zion') {
```

Comment:
Unfortunately this will require additional effort as gem/rspec are not available on the new agent. This is an alternative method to inspect the configuration of the built image: https://sonatype.atlassian.net/l/cp/e2YjPoPe

Reply:
Here's a commit to use the alternative method equivalent to existing RSpec: [5ea7c2f](https://github.com/sonatype/docker-nexus3/pull/157/commits/5ea7c2f6e101d050ad830a4a012c5ff3dd0ee653)
Anyhow we can test it before merging?